test-avocado/lib/host_setup.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
+# -*- coding: utf-8 -*-
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2013 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+
+
+# you have to setup domain in virt-manager default network and reboot the machine to apply changes
+# /etc/libvirt/qemu/networks/default.xml
+# add line
+# <domain name='cockpit.lan' localOnly='yes'/>
+unset command_not_found_handle
+HS_BASE_PCKGS="virt-deploy pystache sshpass telnet fabric python-pip avocado virt-manager qemu-img"
+export HS_GRP="virtualization"
+HS_DEFAULTNET="/etc/libvirt/qemu/networks/default.xml"
+HS_CON="-c qemu:///system"
+
+function host_default_network_domain(){
+    if virsh $HS_CON net-dumpxml default |grep -q "domain name='cockpit.lan'"; then
```

Comment:
it would be nice, but virt-deploy now suppor nowt only "default" network to be used. Thats why I'm doind that.
Second thing is that setup domain in XML for qemu dnsmasq makes test easier (there is properly setup DNS domain delegation and properly set full hostnames)